distribution/test/itests/test-itests-common/src/main/java/ddf/common/test/csw/CswTestCommons.java

```diff
@@ -0,0 +1,146 @@
+            ServiceManager serviceManager) {
+        return getCswSourceProperties(sourceId, CSW_FEDERATED_SOURCE_FACTORY_PID, cswUrl, serviceManager);
+    }
+
+    public static Map<String, Object> getCswConnectedSourceProperties(String sourceId,
+            String cswUrl, ServiceManager serviceManager) {
+        return getCswSourceProperties(sourceId, CSW_CONNECTED_SOURCE_FACTORY_PID, cswUrl, serviceManager);
+    }
+
+    public static Map<String, Object> getCswSourceProperties(String sourceId, String factoryPid,
+            String cswUrl, ServiceManager serviceManager) {
+        Map<String, Object> cswSourceProperties = new HashMap<>();
+        cswSourceProperties.putAll(serviceManager.getMetatypeDefaults(CSW_SOURCE_SYMBOLIC_NAME, factoryPid));
+        cswSourceProperties.put("id", sourceId);
+        cswSourceProperties.put("cswUrl", cswUrl);
+        cswSourceProperties.put("pollInterval", 1);
+        return cswSourceProperties;
+    }
+
+    public static Map<String, Object> getCswRegistryStoreProperties(String sourceId, String cswUrl,
+            ServiceManager serviceManager) {
+        Map<String, Object> cswSourceProperties = new HashMap<>();
+        cswSourceProperties.putAll(serviceManager.getMetatypeDefaults(CSW_REGISTRY_STORE_SYMBOLIC_NAME, CSW_REGISTRY_STORE_FACTORY_PID));
+        cswSourceProperties.put("id", sourceId);
+        cswSourceProperties.put("cswUrl", cswUrl);
+        cswSourceProperties.put("pollInterval", 1);
+        return cswSourceProperties;
+    }
+
+    public static String getCswQuery(String propertyName, String literalValue, String outputFormat,
+            String outputSchema) {
+
+        String schema = "";
+        if (StringUtils.isNotBlank(outputSchema)) {
+            schema = "outputSchema=\"" + outputSchema + "\" ";
+        }
+
+        return getFileContent("/csw-query.xml",
+                ImmutableMap.of("propertyName",
+                        propertyName,
+                        "literal",
+                        literalValue,
+                        "outputFormat",
+                        outputFormat,
+                        "outputSchema",
+                        schema));
+    }
+
+    public static String getCswSubscription(String propertyName, String literalValue,
+            String responseHandler) {
+        return getCswSubscription(propertyName,
+                literalValue,
+                "application/xml",
+                "http://www.opengis.net/cat/csw/2.0.2",
+                responseHandler);
+    }
+
+    public static String getCswSubscription(String propertyName, String literalValue,
+            String ouputFormat, String outputSchema, String responseHandler) {
+
+        String schema = "";
+        if (StringUtils.isNotBlank(outputSchema)) {
+            schema = "outputSchema=\"" + outputSchema + "\" ";
+        }
+
+        return "<csw:GetRecords resultType=\"results\" outputFormat=\"" + ouputFormat + "\" "
```

Comment:
This method was used by other methods so many times with different parameters that it was difficult to create resource files for every instance :/ I would definitely like to untangle it in the follow up ticket